server/models/boxCrime.js

```diff
@@ -0,0 +1,104 @@
+var db = require('../../database/index').db;
+var BoxCrime = require('../../database/models/BoxCrime');
+
+var storeOpenDataInBoxes = (crimeData) => {
+  
+  var asyncBoxStore = crimeData.forEach((crime) => {
+    return new Promise((res, rej) => {
+
+      var long = crime.location.longitude - 0;
+      var lat = crime.location.latitude - 0;
+      var mutilplier = 1000000000000;
+      var boxPadding = 20000000;
+      
+      var coords = {
+        upperLeft: [(((long * mutilplier) - boxPadding) / mutilplier), (((lat * mutilplier) + boxPadding) / mutilplier)],
```

Comment:
( ( long * y ) - z) / y is long - ( z / y )
( ( lat * y ) + z) / y is lat + ( z / y )

if you made a variable for ( z / y ), say offset, these equations could be as simple as
upperLeft: [long - offset, lat + offset] 
upperRight: [long + offset, lat + offset] ... etc